### ISSUE TITLE
In Programmer calculator pane, text is not getting resized for "Hex,Dec,OCT,Bin" value text

### ISSUE THREAD

**Issue by MicrosoftIssueBot** [microsoft/calculator#801]
In Programmer calculator pane, text is not getting resized for "Hex,Dec,OCT,Bin" value text

**Repro steps**
1.Launch the calculator application from start menu.
2.Navigate to  menu navigation button present in left corner if the app and invoke it.
3.Now select the Programmer calculator  and invoke it.
4.Give some input value and observed the resulted Hex,Dec,OCT,Bin value text  size in Textscaling 200%. mode.

**Actual Results**
In programmer calculator pane "Hex, Dec, OCT, Bin value text size is  not enlarged after applying 200 % textscaling.

**Expected Results**
Text size should be increased when Text scaling is applied

**User Impact**
Users with low vision will face difficulty if the text is not getting increased.

![image](https://user-images.githubusercontent.com/1414423/68886736-991b0000-06cc-11ea-9f94-9ead3fd75dbc.png)

**Comment by MicrosoftIssueBot** [comment]
This is your friendly Microsoft Issue Bot. I created this issue automatically as requested by a team member.

**Comment by KareemMAX** [comment]
I think the problem is that the fixed value `CaptionFontSize` is assigned to `ProgModeRadioButtonStyle` ThemeResource.

**Comment by grochocki** [comment]
Hmm, even when specifying a font style like this, the font _should_ scale with the system. I suspect that [IsTextScaleFactorEnabled](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.control.istextscalefactorenabled) might be set to ```false``` somewhere inappropriately (it is intended to be used for non-text, like icons).
